rateLimits/messageRateLimits.md

```diff
@@ -161,7 +195,7 @@ WHILE (retry AND (retries < MAX_RETRIES))
 
 ### Queue Management {#queue-management}
 
-Queue Management requires configuring and maintaining a queue of async operations, http API calls for example, facilitating linear control over when the operation is executed.  The pseudocode below shows a simple queueing solution.  For complex queueing tasks, consider using a queue system such as [RabbitMQ](https://www.rabbitmq.com/) or [Mosquitto](https://mosquitto.org/).   
+Queue Management requires configuring and maintaining a queue of async operations, http API calls for example, facilitating linear control over when the operation is executed.  The pseudocode below shows a simple queueing solution.  For complex queueing tasks, consider using a queue system such as [RabbitMQ](https://www.rabbitmq.com/) or [Mosquitto](https://mosquitto.org/).
```

Comment:
Please only have one space after periods here.

I know this predated your edits, but I'd like to get that fixed for consistency.